### ISSUE TITLE
System.Net.Sockets.SocketException: No such host is known

### ISSUE THREAD

**Issue by jmrados** [Jackett/Jackett#6346]
I am not able to launch Jackett on Windows 10. attached the log file, tried to un-install Jackett, reboot, install latest version all have the same end issue of what is in the log file. 
Tried to search for this issue, which none came up 

(At the bottom of your Jackett Dashboard page) **Jackett version**:  
(if you installed using Jackett.Binaries.Mono.tar.gz) **Mono version**:

-------------------------------
[log.txt](https://github.com/Jackett/Jackett/files/3836706/log.txt)

**Comment by garfield69** [comment]
strange one.

shutdown the service if it isn't already down, and then locate the **ServerConfig.json** file in `C:\ProgramData\Jackett`, open it with a plain text editor (notepad for example) and compare it to mine:
```json
{
  "Port": 9117,
  "AllowExternal": false,
  "APIKey": "r***redactded***5",
  "AdminPassword": null,
  "InstanceId": "i0***redacted***9q",
  "BlackholeDir": "",
  "UpdateDisabled": false,
  "UpdatePrerelease": true,
  "BasePathOverride": "",
  "OmdbApiKey": "",
  "OmdbApiUrl": "",
  "ProxyUrl": "",
  "ProxyType": 0,
  "ProxyPort": null,
  "ProxyUsername": "",
  "ProxyPassword": "",
  "ProxyIsAnonymous": true
}
```
apart from the *APIKey* and *InstanceId* keys, the other settings should look like this.
amend any that are different, save, and start the service back up and see if that helps.

if that does not work, then we may need more detailed enhanced log, so try the following:
1. shutdown the service
2. open a command prompt
3. and cd to `C:\ProgramData\Jackett` and then
4. issue the command `jackettconsole -t -l`
5. and post the **log.txt** here for us to take a look at. 

thanks.

**Comment by jmrados** [comment]
Hi, 
 Once i ran the jackettconsole -t -l and then went into the WebUI, everything worked just fine. Restarted my PC, double clicked on the Jackett icon on my desktop, tried to start the background service again, tried to go into the WebUI and same issue. 
Manually ran jackettconsole and everything works again.

**Comment by garfield69** [comment]
so what you are telling me is that for some reason your jackett service is not starting when you reboot your windows system.

so that is where you need to focus your investigation.
when you installed the windows package as a service, did you use an Administrator login?

check your jackett service in the services panel and compare with mine:
![image](https://user-images.githubusercontent.com/6933240/68713183-27767f80-0602-11ea-8258-44df5101a277.png)
is the path to executable the same, the startup type set to automatic
![image](https://user-images.githubusercontent.com/6933240/68713315-6efd0b80-0602-11ea-93ca-cf240be2831e.png)
is the login set to system account?

**Comment by jmrados** [comment]
Hi, 
 I just want to say sorry for wasting your time. I was a bit pre-occupied with work and did not do the full investigation when I got it to work earlier with your troubleshooting steps.   The issue wound up being my proxy settings that were in the ServerConfig.json file.   When I originally modified it to look like yours, I did all the steps and it still did not work. I rebooted my PC, ran the console command line that you sent, it worked.  I put the file back in, closed out of FireFox, restarted the service and went back in and it was still working. Reboot my pc and for some reason it worked again (possible that I did not overwrite the .json file that had my proxy settings in it?) .   Put everything back the way it was and it started to not work again, then thought that maybe it is the proxy settings.   Removed those again, replaced the file in the programdata directory, restarted Jackett and it worked. Rebooted one more time for S & G's and it is still working. Will look to see if the proxy server address is no longer working for Socks5. 

Again, thank you for your help and sorry for wasting your time on extra crap that I should have tried earlier.
